docker-app/qfieldcloud/core/migrations/0084_project_is_attachment_download_on_demand.py

```diff
@@ -12,9 +12,10 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name="project",
             name="is_attachment_download_on_demand",
+            verbose_name="On demand attachment files download",
             field=models.BooleanField(
                 default=False,
-                help_text="If enabled, it indicates to the client (e.g. QField) that the attachments may be downloaded on demand.",
+                help_text="If enabled, attachment files should be downloaded on demand with QField.",
             ),
         ),
```

Comment:
```suggestion
        migrations.AddField(
            model_name="project",
            name="is_attachment_download_on_demand",
            field=models.BooleanField(
                default=False,
                help_text="If enabled, attachment files should be downloaded on demand with QField.",
                verbose_name="On demand attachment files download",
            ),
        ),
```

`verbose_name` should be within the field itself.

Reply:
TIL